css/style.scss

```diff
@@ -0,0 +1,717 @@
+  li {
+    display: inline-block;
+    width: 64px;
+    height: 63px;
+    border-left: 1px solid #dee0e6;
+    a {
+      display: block;
+      height: 100%;
+      background: url(../img/social_icons.png) no-repeat;
+      background-position: 21px 20px;
+    }
+    &:nth-of-type(2) {
+      a {
+        background-position: -42px 20px;
+      }
+    }
+    &:nth-of-type(3) {
+      a {
+        background-position: -108px 20px;
+      }
+    }
+    &:nth-of-type(4) {
+      border-right: 1px solid #dee0e6;
+      a {
+        background-position: -172px 20px;
+      }
+    }
+  }
+}
+
+header {
+  background: #fff;
+  .container {
+    padding: 60px 0 63px 0;
+    .logo {
+      width: 403px;
+      height: 93px;
+      margin-left: -17px;
+      float: left;
+      a {
+        display: block;
+        height: 100%;
+        background-image: url(../img/logo.png);
+      }
+    }
+    .header-contact {
+      position: relative;
+      float: right;
+      .popup {
+        position: absolute;
+      }
+      ul {
+        margin-top: 67px;
+        margin-right: 12px;
+        li {
+          display: inline-block;
+          height: 24px;
+          width: 67px;
+          a {
+            display: block;
+            box-sizing: content-box;
+            padding: 6px 0;
+            height: 100%;
+            background: url(../img/contact_icons.png) no-repeat;
+            background-position: 18px 0;
+            position: relative;
+            &:hover {
+              background-image: url(../img/contact_icons_hover.png);
+            }
+            &:after {
+              content: "";
+              display: block;
+              width: 66px;
+              height: 24px;
+              position: absolute;
+              top: 6px;
+              border-right: 1px solid #bcbec3;
+            }
+          }
+          &:nth-of-type(2) {
+            a {
+              background-position: -54px 0;
+            }
+          }
+          &:nth-of-type(3) {
+            a {
+              background-position: -128px 0;
+              border: 0;
+              &:after {
+                border: 0;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  nav {
+    border-top: 1px solid #eceef4;
+    .nav-container {
+      width: 1160px;
+      margin: 0 auto;
+      ul {
+        li {
+          display: inline-block;
+          a {
+            @include link_decoration;
+            padding: 36px 20px;
+            font-size: 13px;
+            border-right: 1px solid #eceef4;
+            &:hover {
+              color: $blue;
+            }
+          }
+          .nav-active {
+            background: url(../img/active_nav.png) 50% 100% no-repeat;
+            text-transform: uppercase;
+            color: $blue;
+          }
+          
+        }
+      }
+    }
+  }
+}
+
+.about {
+  background: url(../img/slider_bg.png) no-repeat;
+  -webkit-background-size: 100%;
+  background-size: 100%;
```

Comment:
Will be best practice to create mixin, where you pass a param and apply for `-webkit-background-size`, `background-size: 100%` and others